Report/create_reports.R

```diff
@@ -105,10 +104,22 @@ predictions_cards <- predictions_cards %>%
 class(predictions_cards) <- c("predictions_cards", class(predictions_cards))
 
 print("Saving predictions...")
-saveRDS(predictions_cards,
-  file = prediction_cards_filepath,
-  compress = "xz"
-)
+if (length(signals) == 1) {
+  signal <- signals
+  saveRDS(predictions_cards,
+    file = str_interp(prediction_cards_filepath),
+    compress = "xz"
+  )
+} else {
+  # Save each signal separately.
+  for (signal_group in group_split(predictions_cards, signal)) {
```

Comment:
is the corresponding code to read the saves changed already? I guess that's just line 26 above?

Reply:
We don't need to read it. These files are used as a cache (a year or so ago the pipeline was running out of memory over a long period of time and these files could be read in after an OOM exit to finish the run) + as an input to [Ryan's formatting script](https://github.com/cmu-delphi/forecast-eval/pull/290#issuecomment-1601143737).

It would be better to not separately store the prediction cards and scores, but that would be a separate change.